Source/ACE.Database/Models/Shard/BiotaExtensions.cs

```diff
@@ -1170,9 +1171,18 @@ public static BiotaPropertiesSkill GetOrAddSkill(this Biota biota, ushort type,
                 try
                 {
                     entity = new BiotaPropertiesSkill { ObjectId = biota.Id, Type = type, Object = biota };
-                    biota.BiotaPropertiesSkill.Add(entity);
-                    skillAdded = true;
-                    return entity;
+
+                    if (SkillHelper.ValidSkills.Contains((Skill)type))
```

Comment:
This fix is not appropriate, and will break a ton of functionality.

Almost all monsters in the game will basically have 0 attack skill with this change.